### PR TITLE
PR 2 of 3 (Add 3 and 4 column support in backend + grow and stretch fields)

### PR DESCRIPTION
**Pull request by ghost**
This of one of three PR's I'm writing for Github Issue: https://github.com/octobercms/october/issues/4437

Summary of this PR:

1. Added infomation about the new form columns sizes, created a proper section for it, so people know exactly how to use the new feature.

p.s. Please don't merge until all three PR's are complete.

### CODE REVIEW

**Comment by bennothommo** [comment]
Blocked until PR https://github.com/octobercms/october/issues/4437 is merged.

**Comment by ayumihamsaki** [comment]
This is finished now.

**Comment by ayumihamsaki** [comment]
Not finished - needs updating.

File: backend-forms.md [hunk 1]
```diff
@@ -283,6 +283,16 @@ Option | Description
...
 
+Right now it's not possible to add `Stretch` to YAML for `partial` and `hint` fields, due to the code setup. Instead all you need to do is the following:
+
+    <div class="stretch">
+        ..
+    </div>
+
+Add `stretch` to your class attribute for the container in your hint or partial file and October will do all the rest for you.
+
+I will add a note of this in the doc's.
```
Comment (LukeTowers) [inline backend-forms.md]: remove this line please

Comment (ayumihamsaki) [reply backend-forms.md]: Thanks 👍

**Comment by ayumihamsaki** [comment]
- Need to add the Twig options notes .........

**Comment by ayumihamsaki** [comment]
Finished.

**Comment by ayumihamsaki** [comment]
Finished.

**Comment by ayumihamsaki** [comment]
This is finished - but there are two small things missing:

1. https://octobercms.com/docs/api/backend/classes/formfield#span

Needs changing to:

> Specifies a side. Possible values: auto, left, right, full, third, quarter, two-thirds and three-quarters.

2. https://octobercms.com/docs/api/backend/classes/formfield#line-break

Doesn't exist, needs adding:

> Specifies a line break. Possible values: left, right and none.

**Comment by ayumihamsaki** [comment]
This is finished - but there are three small things missing:

1. https://octobercms.com/docs/api/backend/classes/formfield#span

Needs changing to:

> Specifies a side. Possible values: auto, left, right, full, third, quarter, two-thirds and three-quarters.

2. https://octobercms.com/docs/api/backend/classes/formfield#newline

Doesn't exist, needs adding:

> Specifies a line break. Possible values: true and false.

3. https://octobercms.com/docs/api/backend/classes/formfield#grow

Doesn't exist, needs adding:

> Specifies if this field can have auto height based on content.

**Comment by ayumihamsaki** [comment]
Grow - add `codeeditor`

**Comment by daftspunk** [comment]
We will introduce a simpler approach to this. Thanks for your efforts everyone

**Pull request closed without merging**